### ISSUE TITLE
DTM ratio = 0 and grains=True

### ISSUE THREAD

**Issue by aswinpvijayan** [flaresimulations/synthesizer-grids#21]
The cloudy yaml files have DTM ratio = 0 and grains=True. This is not logically right. The cloudy17 script in synthesizer will treat this scenario as no grains.

e.g. https://github.com/flaresimulations/synthesizer-grids/blob/main/create_grids/cloudy/params/c17.03-sps.yaml#L21

**Comment by stephenmwilkins** [comment]
You're correct, this doesn't conserve things in c17.03. However, it was added for testing purposes.

**Comment by stephenmwilkins** [comment]
Actually, nope that shouldn't be in that file. That should be whatever we decide is the default value for processing SPS models.
